gradle.properties

```diff
@@ -2,7 +2,7 @@ org.gradle.caching=true
 group=io.ballerina.stdlib
 version=2.2.3-SNAPSHOT
```

Comment:
Version should change along with distribution update in toml file

Reply:
updated in [5d3278c](https://github.com/ballerina-platform/module-ballerina-xslt/pull/424/commits/5d3278cdab8a1c88a6a0814a90ea571aebeda4e9)